prime-router/src/main/kotlin/Mappers.kt

```diff
@@ -282,6 +282,18 @@ class Obx8Mapper : Mapper {
     }
 }
 
+class SyntheticDataMapper : Mapper {
+    override val name = "syntheticData"
+
+    override fun valueNames(element: Element, args: List<String>): List<String> {
+        return args
+    }
+
+    override fun apply(element: Element, args: List<String>, values: List<ElementAndValue>): String? {
+        return "PII"
+    }
```

Comment:
I don't understand how this mapper fits into the work of generating synthetic data - can you add a comment?

Reply:
This should have been removed. I missed this during my revert of my work in the translator.